docs/pages/get-started/installation.md

```diff
@@ -23,8 +23,9 @@ Expo CLI is a command line app that is the main interface between a developer an
 ### Recommended Tools
 
 - [VSCode Editor](https://code.visualstudio.com/download)
+  - [VSCode Expo Extension](https://marketplace.visualstudio.com/items?itemName=byCedric.vscode-expo) for app.json debugging and auto complete.
```

Comment:
I think we should link to a docs page about configuring visual studio code for expo development from here rather than inlining a link to expo extension. useful to also suggest the react native extension imo

Reply:
we can add this later